.github/actions/cdk-deploy/action.yml

```diff
@@ -100,4 +100,4 @@ runs:
       working-directory: ${{ inputs.dir }}
       run: |
         echo $STAGE
-        cdk deploy --require-approval never --outputs-file ${HOME}/cdk-outputs.json
+        cdk deploy --require-approval never --outputs-file ${HOME}/veda-backend-cdk-outputs.json
```

Comment:
we're holding this PR until we have the companion change coordinated in veda-deploy, right? I think this is worth a quick shout out in tag up to get the info along to the ghg team as well.

Reply:
Yes, and pending full testing.

> I think this is worth a quick shout out in tag up to get the info along to the ghg team as well.

good idea!

Reply:
Could you explain why this change needs holding? Is there a project expecting the `cdk-outputs.json` file down the line that would break with the rename?

Unless these renames break something, holding seems confusing to me

Reply:
Re-renamed back to `cdk-outputs.json`